### PR TITLE
All inner classes + misc other things.

### PR DESCRIPTION
**Pull request by modmuss50**

### CODE REVIEW

**Review by shartte — APPROVED** [review]

File: mappings/net/minecraft/util/function/CharPredicate.mapping [hunk 1]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5462 net/minecraft/util/function/CharPredicate
```
Comment (liach) [inline mappings/net/minecraft/util/function/CharPredicate.mapping]: Hmm, I recall it is more specific than that in the usage in `Util` class, though i may be wrong

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/state/property/Property.mapping [hunk 2]
```diff
@@ -7,3 +15,17 @@ CLASS net/minecraft/class_2769 net/minecraft/state/property/Property
...
 		ARG 1 value
 	METHOD method_11902 getType ()Ljava/lang/Class;
+	METHOD method_28504 (Ljava/lang/String;)Lcom/mojang/serialization/DataResult;
+		ARG 1 value
+	METHOD method_30041 createValue (Lnet/minecraft/class_2688;)Lnet/minecraft/class_2769$class_4933;
+	METHOD method_30042 createValue (Ljava/lang/Comparable;)Lnet/minecraft/class_2769$class_4933;
+		ARG 1 value
+	METHOD method_30043 stream ()Ljava/util/stream/Stream;
+	METHOD method_30044 getValueCodec ()Lcom/mojang/serialization/Codec;
+	CLASS class_4933 value
```
Comment (YanisBft) [inline mappings/net/minecraft/state/property/Property.mapping]:
```suggestion
	CLASS class_4933 Value
```

File: mappings/net/minecraft/util/function/CharPredicate.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/util/function/CharPredicate.mapping]: Should be in `client.util` imo

Comment (modmuss50) [reply mappings/net/minecraft/util/function/CharPredicate.mapping]: Humm, ill check to see if it is client only, if so ill move it.

Comment (modmuss50) [reply mappings/net/minecraft/util/function/CharPredicate.mapping]: Yes, moved.

**Review by YanisBft — APPROVED** [review]

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping [hunk 3]
```diff
@@ -558,3 +559,4 @@ CLASS net/minecraft/class_4493 com/mojang/blaze3d/platform/GlStateManager
 		FIELD field_5172 x I
 	CLASS class_4534 DstFactor
 	CLASS class_4535 SrcFactor
+	CLASS class_5343 FBOBlitMode
```
Comment (liach) [inline mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:562]: Should've been `FboBlitMode` I guess

**Pull request merged**